src/video/SDL_stretch.c

```diff
@@ -332,17 +332,13 @@ static int scale_mat(const Uint32 *src, int src_w, int src_h, int src_pitch,
     return 0;
 }
 
-#if defined(__SSE2__)
-#define HAVE_SSE2_INTRINSICS 1
-#endif
-
 #if defined(__ARM_NEON)
 #define HAVE_NEON_INTRINSICS 1
 #define CAST_uint8x8_t       (uint8x8_t)
 #define CAST_uint32x2_t      (uint32x2_t)
 #endif
 
-#if defined(__WINRT__) || defined(_MSC_VER)
+#if defined(__WINRT__) || defined(_MSSSC_VER)
```

Comment:
I think this is a typo?

Reply:
Indeed. I fixed it. Thanks for spotting it.